cmd/setup-gh.go

```diff
@@ -132,32 +154,45 @@ func runProviderSetUp(ctx context.Context, sc *providers.SetUpCmd, s spinner.Spi
 	return nil
 }
 
-func getAppName() string {
+func PromptAppName(az providers.AzClientInterface, defaultAppName string) (string, error) {
 	validate := func(input string) error {
```

Comment:
perhaps add the validation for the name here as well?

Reply:
added

Reply:
added